arches/app/templates/views/components/search/map-filter.htm

```diff
@@ -18,102 +20,98 @@
-                    valueAllowUnset: true,
-                    options: spatialFilterTypes,
-                    optionsText: 'title',
-                    optionsValue: 'drawMode',
-                    chosen: {
-                        'width': '100%',
-                        'disable_search_threshold': 10,
-                        'allow_single_deselect': true
-                    }
-                "
-            ></select>
         </div>
+        <!-- /ko-->
 
-    </div>
-    <!-- /ko-->
-
-    <!-- ko if: geoJSONString() !== undefined -->
-    <div class="geojson-card">
-        <div class="geojson-editor" data-bind="codemirror: {
-            value: geoJSONString,
-            mode: { name: 'javascript', json: true },
-            lineNumbers: true
-        }"></div>
-        <!-- ko if: geoJSONErrors().length !== 0 -->
-        <div class="geojson-error-list">
-            <span data-bind="text: $root.translations.geojsonErrors"></span>
-            <ul>
-                <!-- ko foreach: geoJSONErrors() -->
-                <li data-bind="text: message"></li>
-                <!-- /ko-->
-            </ul>
+        <!-- ko if: geoJSONString() !== undefined -->
+        <div class="geojson-card">
+            <div class="geojson-editor" data-bind="codemirror: {
+                value: geoJSONString,
+                mode: { name: 'javascript', json: true },
+                lineNumbers: true
+            }"></div>
+            <!-- ko if: geoJSONErrors().length !== 0 -->
+            <div class="geojson-error-list">
+                <span data-bind="text: $root.translations.geojsonErrors"></span>
+                <ul>
+                    <!-- ko foreach: geoJSONErrors() -->
+                    <li data-bind="text: message"></li>
+                    <!-- /ko-->
+                </ul>
+            </div>
+            <!-- /ko-->
         </div>
         <!-- /ko-->
-        
-    </div>
-    <!-- /ko-->
-    <div class="map-card-zoom-tool">
-        <a href="javascript:void(0);" data-bind="click: function(){
-            editGeoJSON(filter.feature_collection);}">
-            <i class="fa fa-pencil map-card-feature-edit"></i>
-            <span data-bind="text: $root.translations.editGeoJSON"></span>
-        </a>
-        |
-        <a href="javascript:void(0);" data-bind="click: fitToAggregationBounds">
-            <i class="fa fa-search map-card-feature-edit"></i>
-            <span data-bind="text: $root.translations.zoomToAllFeatures"></span>
-        </a>
-        |
-        <a href="javascript:void(0);" data-bind="click: clear">
-            <i class="fa fa-trash map-card-feature-edit"></i>
-            <span data-bind="text: $root.translations.clearFilter"></span>
-        </a>
-
-    </div>
+        <div class="map-card-zoom-tool">
+            <a href="javascript:void(0);" data-bind="click: function(){
+                editGeoJSON(filter.feature_collection);}" aria-label="Edit the geoJSON">
```

Comment:
do we need an aria-label here?  Does the link text not suffice?

Reply:
Fixed in commit 92d7e4a